### PR TITLE
Add API documentation for ARAnchorManager

### PR DESCRIPTION
**Pull request by jsantell**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -8,6 +8,28 @@ Creates an ARView with a [VRDisplay] and a [THREE.WebGLRenderer]. Handles the pa
...
+  });
+```
+
+Currently there's only the `planesupdated` event has an `anchors` property containing an array
+of all updated [THREE.Object3D]s when an anchor update occurs.
+
+### ARAnchorManager THREE.ARAnchorManager#add(object)
+
+Takes a [THREE.Object3D] and sets an anchor at its current position. If anchor is updated, the
+object will be updated accordingly, and a `anchorsupdated` event will be fired on the ARAnchorManager. Returns th ARAnchorManager.
```
Comment (vincentfretin) [inline API.md]: th => the

Comment (jsantell) [reply API.md]: thanks! updated

File: API.md [hunk 2]
```diff
@@ -8,6 +8,28 @@ Creates an ARView with a [VRDisplay] and a [THREE.WebGLRenderer]. Handles the pa
...
+
+Creates an ARAnchorManager with a [VRDisplay]. Allows the adding of [THREE.Object3D]s to be tracked and updated accurately as scene understanding improves via anchors. Extends `EventTarget`, so subscribe to events via:
+
+```js
+  anchorManager.addEventListener('planesupdated', e => {
+    e.anchors.forEach(obj => console.log('updated', obj));
+  });
+```
+
+Currently there's only the `planesupdated` event has an `anchors` property containing an array
```
Comment (judax) [inline API.md]: The event is "anchorsupdated".

**Review by judax — CHANGES_REQUESTED** [review]
Just a minor change as the event name is not "planesupdated" but "anchorsupdated".

File: API.md (hunk 2 above)
Comment (jsantell) [reply API.md]: Ah good catch

**Pull request merged**